cypress/support/pages/signIn.pageObject.js

```diff
@@ -4,15 +4,15 @@ class SignInPageObject extends PageObject {
   url = '/#/login';
 
   get emailField() {
-    return cy.getByDataCy('email-sign-in');
+    return this.getByDataCy('email-sign-in');
```

Comment:
Ensure that the `getByDataCy` method is correctly implemented in the `PageObject` class. This method should return the element with the specified `data-cy` attribute.